examples.md

```diff
@@ -1,44 +1,50 @@
 # Examples
 
-- [C# - NuGet](#c---nuget)
```

Comment:
To be clear, this commit exists because Visual Studio insisted on rewriting the file.

Reply:
I've dropped the commit, but beware that it'll come back the next time someone uses VSC.